src/de/jost_net/JVerein/gui/util/AfaUtil.java

```diff
@@ -319,52 +322,47 @@ private int doAbschreibungAuto(Konto konto, int aktuellesJahr,
     return 1;
   }
   
-  private double getRestwert(Konto konto, ProgressMonitor monitor) throws RemoteException
+  private double getBuchwert(Konto konto, int anschaffungsJahr, int aktuellesJahr, 
+      Geschaeftsjahr aktuellesGJ, ProgressMonitor monitor) throws RemoteException
```

Comment:
`Geschaeftsjahr aktuellesGJ,` ist hier überflüssig, da es nur als `aktuellesGJ.getBeginnGeschaeftsjahr()` verwendet wird,und das ist das gleiche wie `aktuellesJahr`. Also einen der beiden Parameter entfernen

Reply:
aktuellesJahr ist ein int, für beginnGeschaeftsjahr brauche ich ein Date. Ich kann höchstens aktuellesJahr weg machen und es dann in der Methode neu berechnen.

Reply:
oh sorry, da sind die Funktionen sehr ungünstig benannt `getBeginnGeschaeftsjahr() `und `getBeginnGeschaeftsjahrjahr()` da habe ich nicht den Unterschied gesehen, könnte man evtl umbenennen damit das nicht auch anderen passiert.

Reply:
Ja das getBeginnGeschaeftsjahrjahr() hat mich auch schon verwirrt. Ist halt alter Code. Wenn ich wieder mal hier was mache dann benenne ich es um.